bot/exts/backend/sync/_cog.py

```diff
@@ -27,6 +28,11 @@ async def cog_load(self) -> None:
         if guild is None:
             return
 
+        log.info("Waiting for guild to be chunked to start syncers.")
+        while not guild.chunked:
+            await asyncio.sleep(10)
+        log.info("Starting syncers.")
```

Comment:
Is this guaranteed to succeed? If not, there should be some time out and the syncer should still run if possible.

Reply:
The only time this won't succeed is if the member cache isn't populated by Discord.py, which could happen in a very rare case if Discord has gateway issues.

In that case we wouldn't want the user syncer to run at all, as we assume further down if a user isn't in the member cache, they are no longer a member.

Reply:
Added an exception and timeout in acc16545. It will now log the following after 30 minutes:
> discord.ext.commands.errors.ExtensionFailed: Extension 'Sync' raised an error: RuntimeError: The guild was not chunked in time, not loading sync cog.